jmix-charts/charts-flowui-kit/src/main/resources/META-INF/frontend/src/chart/jmix-chart.js

```diff
@@ -74,6 +84,20 @@ class JmixChart extends ResizeMixin(ElementMixin(PolymerElement)) {
         this.initApplicationThemeObserver();
 
         this._root = echarts.init(chart, this.theme);
+
+        this._forwardEvents();
+
+    }
+
+    _forwardEvents() {
+        for (let eventName of JmixChart.forwardedEventNames) {
+            this._root.on(eventName, (params) => {
+                const detail = { ...params, event: undefined };
+                const customEvent = new CustomEvent('jmixchart' + eventName,
+                    { detail: detail });
+                this.dispatchEvent(customEvent);
+            });
+        }
```

Comment:
This method is called only when the chart is initialized.

But after changing the app theme, the chart is recreated, see the `JmixChart._onThemeChange` method.
In this method, all settings are reset to root. Most likely, event dispatching also needs to be reseted.

See video: 

https://github.com/jmix-framework/jmix/assets/75516667/1b7ebf05-e4dc-4b82-b3fb-b1c7556de6fd

Reply:
fixed